src/bananaInBoxRule.ts

```diff
@@ -0,0 +1,76 @@
+import * as Lint from 'tslint';
+import * as ts from 'typescript';
+import * as ast from '@angular/compiler';
+import {BasicTemplateAstVisitor} from './angular/templates/basicTemplateAstVisitor';
+import {NgWalker} from './angular/ngWalker';
+
+
+const BoxInABananaOpen = '([';
```

Comment:
Lets rename it to `ValidSyntaxOpen` or something similar. I'm getting confused reading `BoxInABanana*`, wondering if that was the valid or invalid syntax.